app/scripts/directives/oscRouting.js

```diff
@@ -296,7 +296,7 @@ angular.module("openshiftConsole")
         });
 
         scope.$watch('controls.hideSlider', function(hideSlider){
-          if(!hideSlider && scope.route.alternateServices.length === 1){
+          if(!hideSlider && scope.route.alternativeServices && scope.route.alternateServices.length === 1){
```

Comment:
I think you can 
`if(!hideSlider && _.size(scope.route.alternativeServices) === 1)`